xbmc/Application.cpp

```diff
@@ -2734,11 +2711,42 @@ void CApplication::OnApplicationMessage(ThreadMessage* pMsg)
   break;
 
   case TMSG_LOADPROFILE:
-  {
     CGUIWindowLoginScreen::LoadProfile(pMsg->param1);
     break;
+
+  default:
+    CLog::Log(LOGERROR, "%s: Unhandled threadmessage sent, %u", __FUNCTION__, pMsg->dwMessage);
```

Comment:
if we end up here something really went wrong. in other words an error occurred.